integration_testing/features/coach/coach-lesson-report.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Coach lesson report
+  Coach needs to be able to see the lesson report details
+
+  Background:
+    Given I am signed in to Kolibri as coach user
+      And I am on the *Coach > lessons* page
+      And I see the lesson <lesson_title>
+
+  Scenario: Coach can view the lesson report details
+    When I click the lesson <lesson_title>
+    Then I am on the <lesson_title> page
+      And below I see all the topic <topic>
+    When I click <topic>
+    Then I am on the *lesson report details* page
```

Comment:
Capitalize the words in `*lesson report details*` to reflect the UI:

![2018-08-03_21-35-17](https://user-images.githubusercontent.com/1457929/43662964-f375e734-9767-11e8-8a84-cd70b077d8c9.png)